simulators/active_surface.py

```diff
@@ -44,22 +64,44 @@ def _set_default(self):
         # i.e.: (io_dir[x] = 1)
         self.io_val = [0, 0, 0]
 
-        self.running = False
-        self.delayed_execution = False
-        self.ready = False
-        self.standby_status = True
+        # I/O lines combination that automatically raise the TRIGGER event
+        self.trigger_io_level = [0, 0, 0]
+        self.trigger_io_enable = [0, 0, 0]
+
+        # I/O lines combination that automatically raise the STOP event
+        self.stop_io_level = [0, 0, 0]
+        self.stop_io_enable = [0, 0, 0]
+
+        # I/O lines combination assumed by the driver after a positioning event
+        self.pos_io_level = [0, 0, 0]
+        self.pos_io_enable = [0, 0, 0]
+
+        # I/O lines combination assumed by the driver after positioning to HOME
+        self.home_io_level = [0, 0, 0]
+        self.home_io_enable = [0, 0, 0]
+
+        self.running = False  # True = driver is moving
```

Comment:
What do you think about using ``:`` instead of ``=``?  We save a character in the comment.  For instance: ``# True: driver is moving``.  In case you agree then apply the rule to other comments in this file.